resources/zib/zib-DOSScore.xml

```diff
@@ -0,0 +1,545 @@
+        <discriminator>
+          <type value="type" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.value[x]:valueInteger">
+      <path value="Observation.value[x]" />
+      <sliceName value="valueInteger" />
+      <short value="DOSScoreTotal" />
+      <definition value="Per shift a total score is calculated (minimum 0 and maximum 13). The total scores of three shifts (day, evening and night shifts) are summed to the total score of the day (minimum 0 and maximum 39).&#xD;&#xA;&#xD;&#xA;The DOS scale final score is calculated by dividing the total score of the day by 3 (minimum 0 and maximum 13).&#xD;&#xA;&#xD;&#xA;A DOS scale final score &lt; 3 means that the patient is probably not delirious. A DOS scale final score &gt; 3 means that the patiënt is probably delirious." />
+      <alias value="DOSScoreTotaal" />
+      <type>
+        <code value="integer" />
+      </type>
+      <minValueInteger value="0" />
+      <maxValueInteger value="13" />
+      <mapping>
+        <identity value="zib-dosscore-v1.0-2020EN" />
+        <map value="NL-CM:18.7.3" />
+        <comment value="DOSScoreTotal" />
+      </mapping>
+    </element>
+    <element id="Observation.note">
+      <path value="Observation.note" />
+      <max value="1" />
+    </element>
+    <element id="Observation.note.text">
+      <path value="Observation.note.text" />
+      <short value="Comment" />
+      <definition value="Comment on the DOS score." />
+      <alias value="Toelichting" />
+      <mapping>
+        <identity value="zib-dosscore-v1.0-2020EN" />
+        <map value="NL-CM:18.7.2" />
+        <comment value="Comment" />
+      </mapping>
+    </element>
+    <element id="Observation.component">
+      <path value="Observation.component" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="code" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Observation.component:dozesOff">
+      <path value="Observation.component" />
+      <sliceName value="dozesOff" />
+      <max value="1" />
+    </element>
+    <element id="Observation.component:dozesOff.code">
+      <path value="Observation.component.code" />
+      <patternCodeableConcept>
+        <coding>
+          <system value="http://snomed.info/sct" />
+          <code value="18007006" />
+        </coding>
+      </patternCodeableConcept>
+    </element>
+    <element id="Observation.component:dozesOff.value[x]">
+      <path value="Observation.component.value[x]" />
+      <short value="DozesOff" />
+      <definition value="DOS observation: patient dozes off during conversation or activities.&#xD;&#xA;&#xD;&#xA;Score:&#xD;&#xA;0: never&#xD;&#xA;1: sometimes-always&#xD;&#xA;-: don't know" />
+      <alias value="ZaktWeg" />
+      <type>
+        <code value="integer" />
```

Comment:
Each component should have a minimum and maximum allowed value of 0 and 1 respectively.

Reply:
Done